libs/execution_context/ebpf_core.c

```diff
@@ -448,8 +447,8 @@ _ebpf_core_protocol_map_get_next_key(
     return retval;
 }
 
-static ebpf_result_t
-_ebpf_core_get_next_handle(ebpf_handle_t previous_handle, ebpf_object_type_t type, ebpf_handle_t* next_handle)
+ebpf_result_t
+ebpf_core_get_next_handle(ebpf_handle_t previous_handle, ebpf_object_type_t type, ebpf_handle_t* next_handle)
```

Comment:
Why was this switched from static to exported?

Reply:
it's used in the test code in this PR

Reply:
Why would test code need to enumerate link object but product code won't?

Reply:
I needed to test that the link object is indeed deleted when the program handle and link handle is closed without making an explicit DETACH call.

Reply:
ebpf_object_tracking_terminate already checks that all objects are deleted. 

Related:
Why do we need an enumerate for program and map objects, but don't need one for link objects? For API consistency I think we should have an API to enumerate link objects as well.

Reply:
changed `_ebpf_core_get_next_handle` back to static. `ebpf_object_tracking_terminate()` should catch any leaked objects. Added comment regarding this in the test code.